helm/values.yaml

```diff
@@ -0,0 +1,140 @@
+# ---------------------------------------------------------------------
+# -- Global values
+# ---------------------------------------------------------------------
+image:
+  repository: burningalchemist/sql_exporter
+  pullPolicy: IfNotPresent
+#   tag: $IMAGE_TAG
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+serviceMonitor:
+  enabled: true
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+test:
+  enabled: true
```

Comment:
Is this parameter is resolved somewhere in the template? I didn't find it, but there might be some related logic (due to `helm test` command) behind the flag, that's why I'm asking. 👍

Reply:
It's not, I've forgotten to remove, I'll clean up

Reply:
It should be gone now